### PR TITLE
Related article can return their parent series

### PR DESCRIPTION
**Pull request by eessex**
Updates `relatedArticles` resolver to allow returning the `seriesArticle` that is its parent. Related to routing for Vanguard project.

### CODE REVIEW

File: src/api/apps/graphql/test/resolvers.spec.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ describe("resolvers", () => {
       results.length.should.equal(1)
     })
 
-    it.only("preserves sort order of related articles", async () => {
```
Comment (mbilokonsky) [inline src/api/apps/graphql/test/resolvers.spec.js]: whoops, thanks!

File: src/api/apps/graphql/resolvers.js [hunk 2]
```diff
@@ -271,7 +271,6 @@ export const seriesArticle = root => {
   return new Promise(async (resolve, reject) => {
     const seriesArticles = await promisedMongoFetch({
       layout: "series",
-      published: true,
```
Comment (mbilokonsky) [inline src/api/apps/graphql/resolvers.js]: Help me understand what's happening here - we had initially been saying that `published` was automatically true, but that was a bug? Or this is being removed and appended elsewhere, but I'm not seeing it (it only shows up in the integration mocks?)

Is this change being captured in any tests anywhere? Should we add a third mock to the integration test that isn't published and ensure that it comes through that way, or?

Comment (ashleyjelks) [reply src/api/apps/graphql/resolvers.js]: I think this is b/c we need to return both published and unpublished articles so the bool check is removed. We need to fetch unpublished articles b/c editorial is adding content to the series every day, as such we need to be able to fetch the articles that are still in draft mode in order to render them at the `/series/some-vanguard-slug` route. But @eessex please correct me if I'm wrong!

**Comment by ashleyjelks** [comment]
@eessex lgtm! Merge when you're ready!

File: src/api/apps/graphql/resolvers.js (hunk 2 above)
Comment (eessex) [reply src/api/apps/graphql/resolvers.js]: The articles api does not return unpublished content from users who do not have channel access, so this is extra -- it is hiding unpublished content from admins, making them unable to properly spec drafts.

**Pull request merged**